frontend/src/app.tsx

```diff
@@ -29,17 +31,19 @@ export const App = () => {
       <div className={styles.container}>
         {selectedMessage && (
```

Comment:
Once @mic-smith has done the grid work this will need to be conditional also on there being a report as it will be different for grid. I assume he can do that change later though